### DISCUSSION TITLE
Cannot sponsor as a company

### DISCUSSION THREAD

**Discussion by MootoolsSoftware** [community/community#81014] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,

I try to sponsor a project as a company as shown in the following capture.
Browser: Firefox 120.0.1 (64 bits). Works on Chrome

![image](https://github.com/community/community/assets/15813085/92bd8959-ea51-487c-a461-d889d2cb9abf)

When clicking on Sponsor, I am required to select a region from the dropdown. But the dropdown is empty and no selection can be done from here.
So it makes sponsoring not possible.

Thanks to correct it!

**Comment by deadshot002** [comment]
Eligibility Criteria: GitHub Sponsors or other sponsorship features may have specific eligibility criteria that need to be met. For example, eligibility might be limited to users or organizations in certain countries.

Legal or Compliance Issues: There could be legal or compliance issues preventing sponsorship. Companies might have legal or regulatory constraints that limit their ability to participate in sponsorship programs.

Financial Constraints: A potential sponsor may be facing financial constraints or budget limitations that prevent them from participating in sponsorships at a particular time.

Platform Policies: GitHub may have specific policies or guidelines for sponsorships, and failure to comply with these policies could result in sponsorship-related issues.

Technical Glitches: Sometimes, technical issues or glitches on the platform can impede sponsorship processes. It's advisable to check for any reported issues on GitHub's support or community forums.

**Comment by deadshot002** [comment]
what do use think about it

**Comment by MootoolsSoftware** [comment]
@deadshot002 Sorry, your answer is not appropriate

The region drop-down menu is buggy under Firefox 120, so it's impossible to select a region (whatever the country indicated) and this prevents the form from being validated.
This is not the case with Chrome, where I was able to select my region (Country selected: France) and validate the form
